library/core-utils/src/main/java/im/vector/lib/core/utils/timer/CountUpTimer.kt

```diff
@@ -16,43 +16,39 @@
 
 package im.vector.lib.core.utils.timer
 
-import im.vector.lib.core.utils.flow.tickerFlow
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.cancel
-import kotlinx.coroutines.flow.filter
-import kotlinx.coroutines.flow.launchIn
-import kotlinx.coroutines.flow.map
-import kotlinx.coroutines.flow.onEach
-import java.util.concurrent.atomic.AtomicBoolean
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
 import java.util.concurrent.atomic.AtomicLong
 
 @OptIn(kotlinx.coroutines.ExperimentalCoroutinesApi::class)
-class CountUpTimer(initialTime: Long = 0L, private val intervalInMs: Long = 1_000) {
+class CountUpTimer(
```

Comment:
Since we are not using `tickerFlow` anymore, I think the `@OptIn` can be removed.

Reply:
https://github.com/vector-im/element-android/pull/8058/commits/912c37e3ff5727726bfa03f2494d80add6100047#diff-33cd92dcbf345017e1653f4fe78ab0483508972ba90fc29e8277e8a5e5624d0fR26